### ISSUE TITLE
Provision Jammy VMs

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/figgy#5946]
# Reasoning

Figgy's on Bionic and should be upgraded entirely to Jammy before we can't upgrade packages anymore or upgrading packages breaks our system.

# Boxes to Provision

These names are negotiable, I don't really care what they are.

## Staging
figgy-web-staging1.princeton.edu
figgy-web-staging2.princeton.edu

## Production
figgy-web-prod1
figgy-web-prod2
figgy-web-prod3
figgy-web-prod4

figgy-worker-prod1
figgy-worker-prod2
figgy-worker-prod3
figgy-worker-prod4
figgy-worker-prod5

# Deadline

I don't have a deadline, whatever our risk tolerance is for system upgrades.

# Related PRs

https://github.com/pulibrary/princeton_ansible/pull/3787

**Comment by acozine** [comment]
Current figgy staging boxes have:
2 CPUs 
8 or 16GB memory
Figgy-web-staging-1 has a 30GB hard drive; figgy-staging2 has two hard drives, 40GB + 60GB

I created figgy-web-staging1 and figgy-web-staging2, both in the `Ansible controlled VMs` section. DNS and firewall rules coming soon.

**Comment by acozine** [comment]
Current figgy production boxes have:
figgy1: 4 CPUs 15GB memory 40+60 disk
figgy_web_prod_2: 2 CPUs 8 GB memory, 40+60 disk
figgy3: 2 CPUs 8 GB memory, 40+60 disk
figgy_web_prod_4: 2 CPUs 8 GB memory, 40+60 disk

I created figgy-web-prod1-4, each one has 2 CPUs, 16GB memory, and 30+60 disks.

**Comment by acozine** [comment]
The six current worker boxes for figgy (lib-proc6-11) vary a bit:
Three have 8 CPUs each, three have 2 CPUs each
All six have 32 GB of memory
Five of the six have 40GB + 60 GB of disk, but lib-proc7 has a third disk for 40+60+40

I created figgy-worker-prod1-6, each one has 8 CPUs, 32 GB of memory, and 30 + 60 for disk space.

**Comment by acozine** [comment]
I'm going to close this now. If anybody runs into blockers, we can have a new ticket for those.

**Comment by tpendragon** [comment]
We're reopening these, but just so we can keep track of doing the provisioning of these now that they're made.
